spec/integration_spec.rb

```diff
@@ -1037,13 +1037,13 @@ class SerializedDocument < ActiveRecord::Base
     b = Book.raw_search('hack', { attributesToHighlight: ['*'] })
     expect(b['hits'].length).to eq(1)
     begin
-      expect(b['hits'][0]['name']).to eq('"> hack0r')
+      expect(b['hits'][0]['name']).to eq('"> hack0r').and_raise(StandardError)
       expect(b['hits'][0]['author']).to eq('alert(1)')
       expect(b['hits'][0]['_formatted']['name']).to eq('"> <em>hack</em>0r')
     rescue StandardError
       # rails 4.2's sanitizer
       begin
-        expect(b['hits'][0]['name']).to eq('&quot;&gt; hack0r')
+        expect(b['hits'][0]['name']).to eq('&quot;&gt; hack0r').and_raise(StandardError)
         expect(b['hits'][0]['author']).to eq('')
         expect(b['hits'][0]['_formatted']['name']).to eq('&quot;&gt; <em>hack</em>0r')
       rescue StandardError
```

Comment:
```
Failures:

  1) Book sanitizes attributes
     Failure/Error: Transpec.analyze((expect(b['hits'][0]['name'])), self, "spec/integration_spec.rb_29236_29264", { :expect_source_location => [:context, "method(:expect).source_location"], :expect_example_method_defined_by_user? => [:context, "owner = method(:expect).owner\nowner != RSpec::Core::ExampleGroup &&\n  owner.ancestors.include?(RSpec::Core::ExampleGroup)"] }).to eq('"> hack0r')
     
       expected: "\"> hack0r"
            got: "\"&gt; hack0r"
     
       (compared using ==)
```
It may require further investigation, not sure why this behaviour has changed